### PR TITLE
i3status: add page

### PR DESCRIPTION
**Pull request by marchersimon**
**Version of the command being documented:** i3status 2.13

### CODE REVIEW

File: pages/linux/i3status.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+> Status line for the i3 window manager.
+> This command is usually called from the i3 configuration file.
+> More information: <https://i3wm.org/i3status/manpage.html>.
+
+- Periodically print the status line to stdout, using the default configuration:
+
+`i3status`
+
+- Periodically print the status line to stdout, using a specific consiguration:
```
Comment (navarroaxel) [inline pages/linux/i3status.md:11]: A typo
```suggestion
- Periodically print the status line to stdout, using a specific configuration:
```
What about starts with the `print` verb?
```suggestion
- Print the status line to stdout periodically, using a specific configuration:
```

Comment (marchersimon) [reply pages/linux/i3status.md:11]: I'm not sure, it sounds a bit unnatural. Do you think we should change it?

Comment (navarroaxel) [reply pages/linux/i3status.md:11]: We try to start with a verb :man_shrugging: this example description is like the `watch` page.

File: pages/linux/i3status.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Print the status line to stdout periodically, using the default configuration:
+
+`i3status`
+
+- Print the status line to stdout periodically, using a specific configuration:
+
+`i3status -c {{path/to/i3status.conf}}`
+
+- Display the i3status version and help:
```
Comment (sbrl) [inline pages/linux/i3status.md:15]: i3status is a command name here, so it belongs in backticks.
```suggestion
- Display the `i3status` version and help:
```

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon! Just got 1 quick comment below, but this is otherwise ok.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**